scripts/test-many-configs.sh

```diff
@@ -0,0 +1,327 @@
+#!/bin/bash
+
+
+if [[ ${1} == "" ]]; then
+    echo "Please provide the name of your csv file with the format "URL,SHA,MODULE,tests1-9" on each line."
+    exit
+fi
+
+
+#0. File management/defining all helper functions
+
+flag=0
+scriptDir=$(cd $(dirname $0); pwd)
+csvFile=$(cd $(dirname $1); pwd)/$(basename $1)
+cd ${scriptDir}
+
+if [[ ! -f ${csvFile} ]]; then
+    echo "No such csv exists in the given directory."
+    exit
+fi
+
+if [[ ! -d MC-script-results ]]; then
+    mkdir MC-script-results
+fi
+cd MC-script-results
+
+
+
+function cleanUp() {
+    if [[ ${2} != "" ]]; then
```

Comment:
Give these arguments some local variable names to make it easier to understand what they are supposed to represent.